examples/seismic/acoustic/accuracy.ipynb

```diff
@@ -207,32 +244,56 @@
    "cell_type": "code",
    "execution_count": 9,
    "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Numerical data min,max,abs;    -0.835600    +1.332531    +1.332531\n",
```

Comment:
Maybe printing should be in a vertical way?

Reply:
Not sure what you mean. You mean like:
```
print("min(ref),min(ana); %+12.6f %+12.6f" % (np.min(ref),np.min(ana))
print("max(ref),max(ana); %+12.6f %+12.6f" % (np.max(ref),np.max(ana))
```